starter-code/src/viking.js

```diff
@@ -1,11 +1,75 @@
 // Soldier
-function Soldier() {}
+function Soldier(healthArg, strengthArg) {
+  this.health = healthArg;
+  this.strength = strengthArg;
+}
+
+Soldier.prototype.attack = function() {
+  return this.strength;
+};
+
+Soldier.prototype.receiveDamage = function(damage) {
+  console.log(this.health);
+  console.log(damage);
+
+  this.health = this.health - damage;
+};
 
 // Viking
-function Viking() {}
 
+//constructor function
+function Viking(name, health, strength) {
+  this.name = name;
+  this.health = health;
+  this.strength = strength;
```

Comment:
When making the Viking inherit from the Soldier, the idea is to not write his strength and health characteristics again, but to take them from the Soldier. Here there's only two items, but there can be way more, and they can also be more complicated.
So there's the way to procede:

```javascript
function Viking(name, health, strength) {
  this.name = name;
  Soldier.call(this, health, strength);
}
```

And same for the Saxon :)